app/assets/stylesheets/components/_filter-section.scss

```diff
@@ -0,0 +1,56 @@
+@import "govuk_publishing_components/individual_component_support";
+
+.app-c-filter-section {
+  border-bottom: 1px solid $govuk-border-colour;
+  margin: govuk-spacing(4) 0;
+
+  &:last-child {
+    border-bottom: 0;
+    margin-bottom: 0; 
+  }
+
+  &:first-child {
+    margin-top: 0; 
+  }
+
+  &[open] .app-c-filter-section__summary::before {
+    transform: translateY(70%) rotate(-45deg);
+  }
+}
+
+.app-c-filter-section__summary {
+  display: flex;
```

Comment:
Do we need the `summary-content` block to be separate from the summary itself? Or can we just combine them?

Reply:
Also this probably needs a `cursor: pointer`. And should we think about having a hover state?